### PR TITLE
refactor(rule_engine): refactor doc of save data to timescaledb

### PR DESCRIPTION
**Pull request by Zhiw**

### CODE REVIEW

File: en_US/rule_engine/rule_engine_save_timescaledb.md [hunk 1]
```diff
@@ -37,56 +37,67 @@ Before you start, you need to complete the following operations:
...
-   
-2. Rule Testing
-   Click on Rules on the left menu bar → Rules, click on Create and enter the following rule to match the SQL statement.  In the following rule we read the time `up_timestamp` when the message is reported, the client ID, the message body (Payload) from the `temp_hum/emqx` topic and the temperature and humidity from the message body respectively.
+1. Create TimescaleDB Resource
    
+   Click on `TimescaleDB` under the Data Persistence.
+
+   ![timescaledb](./_assets/timescaledb.png)
+ 
+   Fill in the timescaledb database information you have just created and click `Test`. If there is an error, you should check if the database configuration is correct. Then click on `New` to craete TimescaleDB resource.
```
Comment (Adek06) [inline en_US/rule_engine/rule_engine_save_timescaledb.md:55]: to craete TimescaleDB resource.

`create`

Comment (Zhiw) [reply en_US/rule_engine/rule_engine_save_timescaledb.md]: 已修改

**Review by Adek06 — APPROVED** [review]

**Review by zibuyu1995 — APPROVED** [review]

**Review by wuzufeng — APPROVED** [review]

**Comment by Adek06** [comment]
图 `en_US/rule_engine/_assets/timescaledb_monitor.png` IP 暴露了

**Comment by Zhiw** [comment]
> 图 `en_US/rule_engine/_assets/timescaledb_monitor.png` IP 暴露了

已打码

**Review by chenx-work — APPROVED** [review]

**Pull request merged**